alembic/versions/9d913959375e_migration_message.py

```diff
@@ -0,0 +1,28 @@
+"""Migration message
+
+Revision ID: 9d913959375e
+Revises: 20dd40a4c970
+Create Date: 2019-08-07 08:33:44.620482
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '9d913959375e'
+down_revision = '20dd40a4c970'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('responses', sa.Column('state', sa.Enum('active', 'archived', 'deleted', name='statetype'), nullable=False))
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('responses', 'state')
+    # ### end Alembic commands ###
```

Comment:
What is this alembic file for?

Reply:
Ah, earlier we tried to run migrations with @vic3king that alembic file might have been created that time.

Reply:
You should not push this file @cop1fab